app/templates/emails/brief_clarification_question_confirmation.html

```diff
@@ -4,15 +4,15 @@
     <meta charset="UTF-8">
 </head>
 <body>
-  Dear {{ user_name }},<br />
+  Dear supplier,<br />
   <br />
   Your question about ‘{{ brief_name }}’ has been sent.<br />
   <br />
   You asked:<br />
   <br />
   {{ message }}
```

Comment:
From the story it looks like the `message` here should also be in (typographical) quotes, the same as `brief_name` above.